<launch>
	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
		<param name="resolution" value="0.025" />
		<param name="frame_id" type="string" value="world" />
		<param name="sensor_model/max_range" value="5.0" />
		<remap from="cloud_in" to="/camera/depth_registered/points" />
	</node>
</launch>
